rts/lra/lra-client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -295,7 +295,7 @@ public URI startLRA(URI parentLRA, String clientID, Long timeout, ChronoUnit uni
             }
 
             // validate that there is an LRAInfo response header holding the LRAInfo id
-            Object lraObject = response.getLastHeader(LRA_HTTP_CONTEXT_HEADER);
+            Object lraObject = response.getLocationHeaderAsURI();
```

Comment:
FYI I think the comment about LRAInfo might be slightly out of date

Reply:
Okay, I have just pushed an update.